CoroutineTemplate/domain/build.gradle.kts

```diff
@@ -1,6 +1,8 @@
 plugins {
     id("java-library")
     id("kotlin")
+
```

Comment:
maybe we can just omit this line break 🤨

Reply:
I would like to separate prebuilt and custom plugins, same to what we applied in other modules 🤓 

![image](https://user-images.githubusercontent.com/16315358/163309815-bce62a64-07cb-4f87-ba25-b9bd6e136dcc.png)